inferno-framework/subscriptions-test-kit
# Fi 3007 - subscription validation

# Summary

- Added a method to `subscription_conformance_test.rb` to test if the Provided URL in subscription.criteria is a valid URL and fail the test if it is not.

# Testing Guidance
- Run all rake tasks to confirm they work.
- Run tests suite, confirmed working.
